README.md

```diff
@@ -1,7 +1,16 @@
+ answer-one
+# About-Me
+My name is Faith. I am a medical technologist looking to make a career change! I was first introduced to coding in high school, I didnt take it seriously then, but recently I have found myself gravitating towards it. At my current job we use database softwares that use SQL and I find it very interesting. 
+## For fun
+* Concerts
+* K-drama
+* Foodie
+
 ## Profiles
 * [GitHub](https://github.com/Faith2Faith)
 * [mambowambo.com](https://Mambowambo.com)
 ## Technical Skills
 * Coming soon:
 * Python
 * Javascript 
+ master
```

Comment:
These are tags that git provides when we have a conflict. They are there so we know what changes came from which branch. We'd want to remove those as part of conflict resolution since they aren't part of our final solution. In fact, they could cause code to break if left in there.